aldryn_events/tests/frontend/integration/specs/spec.add.delete.event.js

```diff
@@ -0,0 +1,36 @@
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
+'use strict';
+/* global describe, it, browser */
+
+// #############################################################################
+// INTEGRATION TEST
+describe('Aldryn Events tests: ', function () {
+    var aldrynEventsPage = require('../pages/page.add.delete.event.js');
+
+    it('login to the site with valid username and password', function () {
+        // go to the main page
+        browser.get(aldrynEventsPage.site);
+
+        // click edit mode link
+        aldrynEventsPage.editModeLink.click();
+
+        // wait for username input to appear
+        browser.wait(function () {
+            return browser.isElementPresent(aldrynEventsPage.usernameInput);
+        }, aldrynEventsPage.mainElementsWaitTime);
+
+        // login to the site
+        aldrynEventsPage.loginToSite(
```

Comment:
following the example above, it would be:

```
cmsLogin({
    username: eventsPage.loginUsername,
    password: eventsPage.loginPassword
});
```